packages/dashboard-hardhat-plugin/tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["../../.eslintrc.package.json"]
+}
```

Comment:
Missing new line at end of file

Reply:
New line added in https://github.com/trufflesuite/truffle/commit/4b024e8f12d7ed27835e9c76b47f0f41a6c9f388